.github/workflows/link-check-all.yml

```diff
@@ -0,0 +1,23 @@
+name: Link Check All
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  run:
+    name: Link Check All
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Run Link Check
+      uses: "iterative/link-check.action@master"
+      with:
+        source: "filesystem"
```

Comment:
I would recommend to make `filesystem` the default mode and instead `git diff` can become a special option. This is way more intuitive. 

Anyways, source + filesystem - names are very confusing. It's hard to understand that this simply means that it checks all the files.

Reply:
Fair point, and it matches with the previous usage. This is a remnant of the project originally only having git diff functionality.

Reply:
`filesystem` is now the default mode, and `diff` is a boolean flag